### DISCUSSION TITLE
Liveprog conference

### DISCUSSION THREAD

**Discussion by tluyben** [rabbibotton/clog#370] [General]
Not sure if you are aware, but this conference is very interesting relating to many subjects that are discussed here;

https://liveprog.org/

Relating to CLOG itself, a very dynamic environment, Lisp, which is often used for live programming and moldable development environment, you can see how the criteria on that page look almost written for CLOG taken together with the later discussion topics. 

Thought it would be an interesting topic to add for that reason.

**Comment by rabbibotton** [comment]
I do not have time to write up papers etc (I am a physician, teacher and rabbi outside of programming), but happy to do demos and speak when asked even to travel sometimes willing. If they would like to see some live demos or someone speak about the power of Live programming no one beats a Lisp guy :) and no Lisp guy beats a CLOG Lisp guy.

**Reply by rabbibotton** [reply]
Also if anyone ever wants to write papers, demos and speak I will aways support them.

**Reply by tluyben** [reply]
Well, I also added it to get ideas from the academic world: many gifted people are presenting and giving workshops there. Also the other way around; many young talents have probably heard of Lisp but never seen the power of it.

**Comment by rabbibotton** [comment]
I submitted the Master Class 1 to them as a perfect fit for them.

**Reply by rabbibotton** [reply]
I probably can't make it though as the conference is TBA so far on a major Jewish holiday. Maybe can speak remote or send a video.
